client/src/components/profileDetails/PrimaryProfile.tsx

```
@@ -0,0 +1,87 @@
+  const handleNavigate = () => {
+    navigate(`/${linkurl.toLowerCase().replace(" ", "/")}`);
+    onComplete();
+  };
+
+  return (
+    <div className="mt-4 bg-gray-100 p-4 rounded-md border-2 border-green-800">
+      <p className="mt-4 text-lg sm:text-2xl text-green-900 font-bold">{msgData}</p>
+      <button
+        onClick={handleNavigate}
+        className={`font-semibold inline-flex items-center gap-2 mt-2 text-white hover:text-gray-200 transition text-sm sm:text-lg py-2 px-4 rounded-md ${isGreen ? 'bg-green-800' : 'bg-indigo-900'}`}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)